ui/src/reusable_ui/components/wizard/test/__snapshots__/WizardStep.test.tsx.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`WizardStep matches snapshot 1`] = `
-<React.Fragment>
+<Fragment>
```

Comment:
style thing only ... can we use the empty tag notation here?

Reply:
This stuff is autogenerated by some combination of Enzyme/Jest/Enzyme plugins. But I agree in our source code I like the `<>` notation